CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+*       @confidential-containers/documentation-maintainers
```

Comment:
So if we want this to work we will need to give the documentation maintainers group special permissions on this repo via the settings panel. I can do that, but maybe we should just get rid of the documentation maintainers group since it doesn't have very many people in it.

Reply:
I guess a philosophical question, I was going for get the code/documentation in then fix links etc afterwards , outlined things to address in #83 in this [comment](https://github.com/confidential-containers/community/issues/83#issuecomment-1584369853)
Appreciate I completely failed to link this PR to that issue (fixed that now).

@fitzthum Would you be OK if I copy your points from this PR into the issue as needing addressed, we merge this PR and I create a follow on PR(s) to fix things up such as references/links etc?

Reply:
I don't see any benefit to doing it separately unless the fixes are going to add a huge delay.